test/e2e/cli_test.go

```diff
@@ -1014,6 +1014,48 @@ func (s *CLISuite) TestRetryOmit() {
 		WaitForWorkflow(20 * time.Second)
 }
 
+func (s *CLISuite) TestResourceTemplateStopAndTerminate() {
+	s.testNeedsOffloading()
```

Comment:
why does this need offloading?

Reply:
It needs for CI. The current CI is running with PROFILE=mysql. This Test will be executed part of `CLIWithServerSuite`

Reply:
sorry - my point is that nothing in this test indicates it needs offloading to pass - if you remove this like, does it run?

Reply:
No, CLI GET command will return error `offload node status is not supported`. Current CI ENV is enabled DB and `ALWAYS_OFFLOAD_NODE_STATUS=true` for mysql.

Reply:
ah - ok - I get it